docs/contributing/database-migrations/mssql.md

```diff
@@ -11,97 +11,72 @@ For instructions on how to apply database migrations, please refer to the
+   with the current date.
 
-### Data migration
+Please take care to ensure:
 
-We now need to write a script that migrates any data from the old location to the new locations.
-This script should ideally be written in a way that supports batching, i.e. execute for X number of
-rows at a time. This helps avoiding locking the database. When running the scripts against the
-server please keep running it until it affects `0 rows`.
+- any existing stored procedure accepts the same input parameters and that new parameters have
+  nullable defaults
+- when a column is renamed the existing stored procedures first check (coalesce) the new location
+  before falling back to the old location
+- continued updating of the old data columns since in case of a rollback no data should be lost
 
-### Non-backwards compatible
+### Non-backwards compatible migration
 
-These changes should be written from the perspective of "all data has been migrated". And any old
-_Stored Procedures_ that were kept around for backwards compatibility should be removed. Any logic
-for syncing old and new data should also be removed in this step.
+These changes should be written from the perspective of "all data has been migrated" and any old
+stored procedures that were kept around for backwards compatibility should be removed. Any logic for
+syncing old and new data should also be removed in this step.
 
-Since `Sql/dbo` represents the current state we need to introduce a "future" state which we will
-call `dbo_finalization`.
-
-1. Copy the relevant `.sql` files from `src/Sql/dbo` to `src/Sql/dbo_finalization`.
-2. Remove the backwards compatibility which is no longer needed.
-3. Write a new Migration and place it in `src/Migrator/DbScripts_finalization`, name it
+1. Remove the backwards compatibility that is no longer needed.
+2. Write a new Migration and place it in `src/Migrator/DbScripts_finalization`. Name it
    `YYYY-0M-FinalizationMigration.sql`.
    - Typically migrations are designed to be run in sequence. However since the migrations in
-     DbScripts_future can be run out of order, care must be taken to ensure they remain compatible
-     with the changes to DbScripts. In order to achieve this we only keep a single migration, which
-     executes all backwards incompatible schema changes.
-
-### [Not Yet Implemented] Manual MSSQL migrations
-
-There may be a need for a migration to be run outside of our normal update process. These types of
-migrations should be saved for very exceptional purposes. One such reason could be an Index rebuild.
-
-1. Write a new Migration with a prefixed current date and place it in
-   `src/Migrator/DbScripts_manual`
-2. After it has been run against our Cloud environments and we are satisfied with the outcome,
-   create a PR to move it to `DbScripts`. This will enable it to be run by our Migrator processes in
-   self-host and clean installs of both cloud and self-host environments
```

Comment:
Have we determined we will never do this?

Reply:
I'd say yes. It's problematic for self-host. Things like this just aren't added to source control if they have no auditable change. A rebuild could be done, but that isn't necessary in the source as it is transient.